\chapter{Discussion}

In this section, I will do an integrative discussion of the different manuscripts that compose this doctoral thesis. Here, my objective was not to substitute but to introduce and complement the discussion of each one of the manuscripts that are included in \autoref{chap:appendix}. First, I will review the projects that were conducted using the research framework to study the Johns Hopkins longitudinal cohort of myositis patients. Then, I will comment on the different studies that were completed using the RNAseq profiling of myositis muscle biopsies.

As it was mentioned earlier, given the uncertainty regarding the etiology of the different types of myositis, we believed that it was necessary to approach studying the classification and its pathogenesis in parallel. This way we would be certain to be studying homogeneous groups of patients.

It may be argued that the clinical section of this doctoral dissertation does not fall within the realm of bioinformatics. However, bioinformatics was originally defined as the study of informatic processes in biotic systems.\cite{Hogeweg1978} It is a broad multidisciplinary field that combines biology, computer science, information engineering, mathematics, and statistics to analyze and interpret biological data. Accordingly, the informatic methodology that we have used to retrieve, organize, analyze, and synthesize efficiently the longitudinal clinical information of our myositis patients fits the original definition of what is bioinformatics.

Particularly, this doctoral dissertation would be a good example of research in translational bioinformatics, a young discipline defined as "the development of storage, analytic, and interpretive methods to optimize the transformation of increasingly voluminous biomedical data, and genomic data, into proactive, predictive, preventive, and participatory health".\cite{Tenenbaum2016} Particularly, we are using a combination of clinical and basic bioinformatic research to get a deeper understanding on various aspects of a disease, including diagnosis, classification, therapeutics, prognosis and pathophysiology of different types of myositis.

\section{Longitudinal cohort studies of myositis subsets}

Arguably, the main methodological advantage of cohort studies over other types of observational studies is that they allow exploring multiple exposures in different subsets of patients. Alternatively, they usually require a costly infrastructure and a considerable investment of time to obtain consistently high-quality data over a prolonged period of time. In all the studies that are included in this section, we show the full power of combining a relatively well-managed cohort of patients with an efficient data management and data analysis research framework.

By planning complex projects (e.g. comparing the performance of \gls{msa} to the 2017 EULAR/ACR criteria) as a collection of smaller and more manageable studies (e.g. anti-Mi2 study) we were able to gradually explore the best data-gathering strategies, analytical tools, and inherent limitations of our cohort, progressively adapting and incrementally increasing the quality and efficiency of our studies over time.

Thanks to this stepwise approach to large epidemiologic projects we were able to analyze the clinical features, survival, association with cancer, prognosis of particular \gls{msa} (e.g. anti-Mi2), and both disease-specific (e.g. anti-U1RNP) and non-disease-specific (anti-Ro52) \gls{maa}. Moreover, we were able to systematically compare the effectiveness of certain treatments within prevalent myositis subsets (e.g. \gls{mtx} vs. \gls{aza} in the \gls{as}). Finally, we could aggregate and expand the data of various studies to answer relevant questions to our whole are or research (e.g. comparing the performance of \gls{msa} to the 2017 EULAR/ACR criteria).

First, regarding the analysis of \gls{msa} it is important to emphasize that these autoantibodies are generally mutually exclusive. This implies that each one of them can be compared with the other with very little concern for patients being included simultaneously in more than one category. In the study analyzing the clinical phenotype of anti-Mi2 autoantibodies (Appendix \autoref{sec:mi2_clinical}) we used this characteristic to our advantage comparing a group of 58 anti-Mi2 patients with other relevant autoantibody-defined groups including: (1) non-Mi2 \gls{dm} comprising the anti-NXP2, anti-TIF1$\gamma$, and anti-MDA5; (2) \gls{as} including anti-Jo1, anti-PL7, and anti-PL12], and (3) \gls{imnm} including anti-SRP and anti-HMGCR antibodies.

With this design, we could define that patients with anti-Mi2 autoantibodies have more weakness and higher CK levels than patients with anti-MI2 negative DM. Also, we could establish that the muscle involvement was more severe in patients with anti-Mi2 autoantibodies than in the \gls{as} and similar in the upper extremities to \gls{imnm}. In contrast with the frequent and severe muscle disease, we found that the extramuscular manifestations were less common in anti-Mi2 patients. Moreover, we could not find a significant association with cancer compared to the general population. Importantly, our data suggested that the levels of anti-Mi2 autoantibodies were associated with the enzyme levels and the strength and that it decreased with treatment, occasionally normalizing. This last finding may be of practical importance since it suggests that anti-Mi2 autoantibodies may be useful not just as a useful biomarker of disease activity but to decide whether it is safe to completely withdraw immunosuppressive medication from patients when the anti-Mi2 levels decrease.

Besides, multiple serologic techniques can be used to determine anti-Mi2 autoantibodies. A key part of our work on defining the characteristics of specific autoantibody-defined subsets of patients is to validate the serologic techniques that we are using so we can know how if they are reliable and, if they are not, how to modify the manufacturer's recommendations to be used confidently. In this case, we validated the EUROLine Myositis Profile 4 line blot by immunoprecipitation and ELISA (Appendix \autoref{sec:mi2_serologic}) finding that only those subjects that are positive for both anti-Mi2$\alpha$ and anti-Mi2$\beta$ can be reliably considered anti-Mi2 without further validation. Alternatively, those who are positive just for anti-Mi2$\beta$ are usually false positive and those who are only positive for anti-Mi2$\alpha$ have only about 50\% chance of being true anti-Mi2 positives.

As it was mentioned in the introduction, the current terminology of \gls{msa} groups together two different populations of autoantibodies, those that are not specific for any given phenotype and those that, despite being associated with a homogeneous phenotype, show features that are found in non-myositis autoimmune diseases. Relevant to the bioinformatic analysis of these groups of patients, those that are specific for a homogeneous phenotype and mutually exclusive can be analyzed similarly to \gls{msa} groups of patients, but those that are not specific for any given phenotype have to be analyzed by comparing \gls{maa}-positive with \gls{maa}-negative patients both globally and for each of the different subgroups. In this doctoral thesis, we studied anti-U1RNP patients as an example of a \gls{maa} that is associated with a particular phenotype (Appendix \autoref{sec:u1rnp}). By analyzing a cohort of 20 anti-U1RNP patients we found that these individuals typically present with proximal weakness and necrotizing muscle biopsies, showing arthritis, dermatitis, and ILD as the most common extramuscular clinical features. Also, pericarditis and glomerulonephritis were uniquely found in patients with coexisting anti-U1RNP autoantibodies and anti-Ro52 autoantibodies.

We also used our research framework to study anti-Ro52 autoantibodies in a cohort of 371 juvenile \gls{dm} (Appendix \autoref{sec:ro52}). These autoantibodies are \gls{maa} that are not specific of any particular phenotype but have been suggested to act as disease modifiers. In our study we found that anti-Ro52 autoantibodies are present in 14\% of patients with juvenile myositis and are strongly associated with anti-MDA5 and \gls{as} autoantibodies. Also, we determined that in patients with juvenile \gls{dm}, those with anti-Ro52 autoantibodies were more likely to have ILD, had more severe disease and poorer prognosis.

Finally, by aggregating and expanding the data of various of our earlier studies we were able to summarize the phenotype of 524 \gls{msa}-positive patients using factor analysis of mixed data and demonstrate the utility of \gls{msa} to subclassify myositis patients (Appendix \autoref{sec:atb_class}). Moreover, \gls{aic} and \gls{bic} of the different regression models showed that \gls{msa} outperform the 2017 EULAR/ACR criteria to predict the factor-analysis-derived phenotype in adult \gls{msa}-positive myositis patients. Based on our results we proposed a classification (Table \ref{tab:criteria}) using the \gls{msa} to inform patient selection for assembling myositis cohorts of the most phenotypically and clinically homogeneous groups.

Importantly for this study is the methodology that we used to handle the phenotype of the patients statistically. In other projects, the authors defined questionable homogeneous clusters of patients linking them to clinical classifications based on the opinion of experts.\cite{Mariampillai2018} Alternatively, in our study we used the clinical characteristics of the patients to answer a relevant clinical question: do the autoantibodies outperform clinical classifications systems in myositis? The rationale behind this question is that if the target of the immune response is defined by the autoantigen, in those diseases with a humoral response, the autoantibodies will completely define the disease. Therefore, there will be no possible combination of clinical features that will be able to outperform the autoantibodies to predict the phenotype of the patients.

\begin{table}
\caption{Proposal of myositis classification based on myositis-specific autoantibodies}
\label{tab:criteria}
\resizebox{\textwidth}{!}{
	\begin{tabular}{l|l}
		\multirow{6}{*}{Myositis-specific autoantibody + } & Muscle weakness or           \\
		                                                   & Creatine kinase elevation or \\
		                                                   & Interstitial lung disease or \\
		                                                   & Arthritis or                 \\
		                                                   & Gottron's sign or papules or \\
		                                                   & Heliotrope
	\end{tabular}}
\end{table}

\section{Transcriptome profiling of myositis muscle biopsies}

The sheer amount of information that we obtained by performing RNAseq in the muscle biopsies of patients with different types of inflammatory myopathy has the potential to answer multiple questions relevant to the pathogenesis of myositis simultaneously. In this doctoral thesis, we show the results of addressing three specific issues. First, we wanted to explore the expression of the different autoantigens in patients with different autoantibodies; second, we did an in-depth study of the interferon pathway (both type I and type II) in the different clinical and autoantibody myositis groups; finally, we developed a methodology to identify unique gene expression profiles in each clinical and autoantibody myositis group.

Regarding the first of the studies,(Appendix \autoref{sec:autoantigens}) after the first \gls{msa} were discovered in myositis, it was proposed that regenerating muscle cells in biopsy tissue from human myositis muscle express high levels of several myositis autoantigens, including Mi2, TIF1$\gamma$, Jo1, HMGCR, and SRP.\cite{Mohassel2015,Mammen2009,CasciolaRosen2005,Allenbach2018} Given this observation, it has been proposed that an increased expression of myositis autoantigens may initiate and/or maintain autoimmunity against these particular proteins. However, it has not been determined if autoantigens other than Mi2, TIF1$\gamma$, Jo1, HMGCR and SRP are expressed at high levels in regenerating muscle, if autoantigen expression patterns differ between myositis subgroups, or if there is a relationship between the expression level of an autoantigen and the presence of its corresponding autoantibody. Thus, we explored these questions in our dataset finding that myositis autoantigens are in fact highly expressed during muscle regeneration, however, we could not find any significant association between the increased expression of a given autoantigen and its corresponding autoantibody. This is, patients with anti-HMGCR antibodies did not show higher levels of HMGCR than patients with anti-SRP antibodies, and so on.

As per the second study using these data,(Appendix \autoref{sec:ifn}) prior studies had established the preferential activation of the \gls{ifn}1 pathway in \gls{dm} muscle.\cite{Greenberg2005} However, activation of the \gls{ifn}1 pathway has not been compared between patients with \gls{dm} with different \gls{msa}. Furthermore, the \gls{ifn}1 pathway activation was found to be relatively low in \gls{ibm} but has not been systematically explored in \gls{as} or \gls{imnm}.\cite{Greenberg2005,Salajegheh2010,Greenberg2002} Similarly, although \gls{ifn}2 pathway activation has been implicated in IBM muscle,\cite{Ivanidze2011,Allenbach2014} activation of \gls{ifn}2 pathways in muscle biopsies from patients with \gls{imnm}, \gls{as}, and \gls{ibm} has not been systematically analyzed. Our study confirms that \gls{dm} muscle biopsies are characterized by high levels of both \gls{ifn}1- and \gls{ifn}2-inducible genes. In contrast, biopsies from patients with \gls{as} and \gls{ibm} reveal gene expression patterns consistent with prominent \gls{ifn}2 activation. Finally, RNA sequencing analysis reveals that \gls{imnm} biopsies show relatively low activation of the \gls{ifn} pathway. Moreover, all the different \gls{msa} \gls{dm} groups that we explored showed a similar level of activation of both \gls{ifn} pathways.

Finally, microarray analysis led to the discovery that type I and type II \gls{ifn}-inducible genes are upregulated in muscle biopsies from patients with \gls{dm}\cite{Greenberg2005} and \gls{ibm},\cite{Ivanidze2011,Allenbach2014} respectively. However, disease-specific gene expression profiles have not been fully described in patients with \gls{imnm}, \gls{as} or any of the \gls{msa}-defined subtypes of \gls{dm}. Furthermore, little attention has been given to genes that are differentially expressed between patients with different types and subtypes of myositis.\cite{Greenberg2005,Greenberg2002,Hamann2017,Raju2005} In this last study included in the doctoral thesis,(Appendix \autoref{sec:rnaseq_ml}) we trained machine learning algorithms to classify muscle biopsies using transcriptomic data from normal, \gls{ibm} and \gls{msa}-positive muscle biopsies. We then used recursive feature elimination to identify novel disease-specific gene expression patterns that may be pathologically relevant in \gls{dm},  \gls{as}, \gls{imnm}, \gls{ibm} and \gls{msa}-defined subtypes of myositis. With this approach we could determine that \gls{dm},  \gls{as}, \gls{imnm}, and \gls{ibm} are best distinguished based on their gene expression pattern by using linear support vector machines. Furthermore, by applying recursive feature elimination to these classification models, we not only confirmed known pathological pathways in myositis, such as the role of type I \gls{ifn} in \gls{dm}, we also identified novel genes that are uniquely upregulated in other types and \gls{msa}-defined subtypes of myositis.

This study contained results of key biological and clinical importance but also relevant information for future bioinformatic studies in the field. Most importantly, it defined that the linear support vector machines outperform other models that theoretically should behave well in scenarios with high dimensionality and low sample size, like the random forests. Also, we were able to validate the recursive feature elimination to sort the different genes in order of importance. Finally, we compared different strategies to restrict the number of genes determining that doing it before the cross-validation is equivalent and less computationally expensive than doing it afterward.

\section{Limitations and future directions}
Our studies suggest that autoantibodies outperform clinical classification systems to predict the phenotype of myositis patients and found a variety of specific pathogenic pathways, and clinical associations. Notwithstanding this, it will be necessary to validate our results in different cohorts and using different techniques to confirm them and understand better its importance. Also, some of these studies have been the largest ever conducted in our field, including often samples from different epidemiologic backgrounds. However, the sample size in some of our autoantibody groups was limited and we may have lost relevant signals for this reason. Future efforts will aim to increase the sample size of these patient groups. Finally, our clinical studies used retrospectively collected clinical data, and our transcriptional research focused on studying muscle tissue. Prospectively, it will be key to expand the clinical information that we collect and to study other biological myositis samples such as skin, lung, or blood cells.

At this moment we are conducting several studies to validate particular transcriptional signals that we detected in our studies at the protein level. Moreover, international collaborative efforts are being conducted to validate the importance of autoantibodies in myositis classification through the International Myositis Assessment and Clinical Studies Group and the \gls{enmc}.

Also, we are conducting single-cell and single-nuclei RNAseq studies in myositis muscle biopsies to understand which cells in the muscle are expressing each one of the specific markers that we detected. Moreover, we are also analyzing the distribution of the expression of the different markers is myositis muscle biopsies using spatial transcriptomics. Besides, we are studying the transcriptome of blood cells at a single-cell resolution to understand the distinct inflammatory response of each one of the myositis subgroups.

Finally, any of the classical types of myositis and myositis-overlap syndromes that we have studied have a clear etiology. However, there are types of inflammatory myositis, like checkpoint-inhibitor-induced or graft vs. host myositis that have a known cause and may help us to understand the rest. We are establishing collaborations with experts all around the world to study these types of man-made myositis.

 